numpy/core/numeric.py

```diff
@@ -973,7 +973,8 @@ def tensordot(a, b, axes=2):
 
     When `axes` is integer_like, the sequence for evaluation will be: first
     the -Nth axis in `a` and 0th axis in `b`, and the -1th axis in `a` and
-    Nth axis in `b` last.
+    (N-1)th axis in `b` last. When N-1 is smaller than 0, or when -N is larger
+    than -1, the element of 'a' and 'b' are defined as the 'axes'.
```

Comment:
I think this whole section should be removed. It is less clear than the explanation above (in the `Parameters` section).

Reply:
I also think this part is a little redundant.
However, I think this part also has certain significance.
1. I think it should be claimed what would happen when N is less than 0, because there is no limitation or error warning when N < 0.
2. the orders of a and b axes are important and it's not clear in the parameters section.
Therefore, I add some details to this part to make it clearer and more meaningful.